From: invakid404 <invakid404@riseup.net>
Date: Mon, 22 Jan 2021 20:38:21 +0200
Subject: Fix type length limit compile issue

The Rust compiler complains about the type_length_limit of spotify_tui,
so I created this patch to address the issue.

---

diff --git a/src/main.rs b/src/main.rs
index adb0de5..59ac2ae 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -1,3 +1,4 @@
+#![type_length_limit="3576009"]
 mod app;
 mod banner;
 mod cli;
